superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -356,6 +361,12 @@ const StyledNotificationAddButton = styled.div`
   }
 `;
 
+const StyledNotificationMethodWrapper = styled.div`
+  .input-container {
+    margin-left: 0 !important;
```

Comment:
can we do this without an important?

Reply:
Sure can, thanks for this catch! Added specificity instead of !important in [`this commit`](https://github.com/apache/superset/pull/22709/commits/74433c1059395aec4a1b4631b5578dd8231b7de3).